src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={`${movie.imgUrl}`}
```

Comment:
better use destructurisation to not repeat `movie.` every time.
And you don't need backsticks here)
```suggestion
          src={movie.imgUrl}
```